src/app/components/bitcoin-fees-list/components/fees-list-item.tsx

```diff
@@ -32,6 +31,7 @@ export function FeesListItem({
       px="space.04"
       py="space.06"
       transition="transition"
+      type="button"
```

Comment:
It's a bit late to consider this now, but when we start on the UI library lets make a generic `Button` / update `LeatherButton` so we can replace all these `styled.button` and enforce `type=button` as a standard

Reply:
Yep, agree!

Reply:
https://github.com/leather-wallet/extension/issues/4523